### PR TITLE
initial drop-ins docs

### PR DESCRIPTION
**Pull request by spencerhunter**

### CODE REVIEW

File: source/includes/_dropIns.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+#### HTTP request
+
+`POST https://api.dwolla.com/client-tokens`
+
+
+#### Request parameters
+
+| Parameter | Required | Type | Description |
+|-----------|----------|----------------|-------------|
+| action | yes | object | A granular permission for the Customer perform an action within a drop-in component. [Reference the client token actions to learn more](#client-token-actions). |
```
Comment (ShreyaThapa) [inline source/includes/_dropIns.md:33]: > A granular permission for the Customer perform an action within a drop-in component.

perform needs to be "performing"

File: source/includes/_dropIns.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+| _links | yes | object | A _links JSON object that contains a link the desired `customer` to perform the action within the drop-in component. |
```
Comment (ShreyaThapa) [inline source/includes/_dropIns.md:34]: >A _links JSON object that contains a link the desired `customer` to perform the action within the drop-in component.

Needs to be:

"A _links JSON object that contains a link to the desired `customer` performing the action within the drop-in component."

File: source/includes/_dropIns.md (hunk 1 above)
Comment (spencerhunter) [reply source/includes/_dropIns.md:33]: good catch. fixing

Comment (misskellymore) [inline source/includes/_dropIns.md:33]: "A granular permission for the Customer performing"

File: source/includes/_dropIns.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+  },
+  :action => "customer.update"
+}
+
+client_token = app_token.post "client-tokens", request_body
+client_token.token # => "4adF858jPeQ9RnojMHdqSD2KwsvmhO7Ti7cI5woOiBGCpH5krY"
+```
+```php
+/**
+ * No example for this language yet.
```
Comment (misskellymore) [inline source/includes/_dropIns.md:73]: I can write a ticket for this and figure it out later if you'd like

File: source/includes/_dropIns.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+| Parameter | Required | Type | Description |
+|-----------|----------|----------------|-------------|
+| action | yes | object | A granular permission for the Customer perform an action within a drop-in component. [Reference the client token actions to learn more](#client-token-actions). |
+| _links | yes | object | A _links JSON object that contains a link the desired `customer` to perform the action within the drop-in component. |
+
+#### Request and response
+
+```raw
+POST https://api-sandbox.dwolla.com/client-tokens 
+Accept: application/vnd.dwolla.v1.hal+json Content-Type: application/json
```
Comment (ShreyaThapa) [inline source/includes/_dropIns.md:40]: >Content-Type: application/json

This should probably be in a new line.

**Review by ShreyaThapa — APPROVED** [review]

**Pull request merged**